src/auth/auth.controller.spec.ts

```diff
@@ -77,6 +108,35 @@ describe("AuthController", () => {
           });
         });
     });
+
+    it("should return a token and 201 http code when account information is correct.", async () => {
+      const expectedResult = {
+        accessToken: "mocked_token",
+        statusCode: HttpStatus.CREATED,
+      };
+
+      jest
+        .spyOn(localStrategy, "validate")
+        .mockResolvedValue({ email: "test@example.com", id: 1 });
+
+      const result = await authController.login(localStrategy);
+
+      expect(result).toEqual(expectedResult);
+    });
+
+    it("should return Unauthorized and 401 http code when account information is failure.", async () => {
```

Comment:
grammer recommend:

- ...when account information is not correct
- ...when account information is wrong
- ...when failed to verify account information
  - this may not accurate, cuz it might means that the system failed to execute the verify instead of it verified it is wrong.

Reply:
I think this closer to my original intention.
- ...when account information is wrong
  - 當帳號資訊錯了的時候


[refactor: revised grammar questions about words](https://github.com/Cophr/backend/pull/10/commits/7044fff2e4c62114fc350e06818e60240d7ad97e)